src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/JdepsMerger.kt

```diff
@@ -64,9 +83,33 @@ class JdepsMerger: CommandLineProgram {
       BufferedOutputStream(File(output).outputStream()).use {
         it.write(rootBuilder.build().toByteArray())
       }
+
+      if (reportUnusedDeps != "off") {
+        val unusedLabels = dependencyMap.values
+          .filter { it.kind == Deps.Dependency.Kind.UNUSED }
+          .mapNotNull { readJarOwnerFromManifest(Paths.get(it.path)).label }
+          .filter {it != label}
+
+        if (unusedLabels.isNotEmpty()) {
+          val open = "\u001b[35m\u001b[1m"
+          val close = "\u001b[0m"
+          val command =
+            """
+            |$open ** Please remove the following dependencies:$close ${unusedLabels.joinToString(" ")} from $label 
+            |$open ** You can use the following buildozer command:$close buildozer 'remove deps ${unusedLabels.joinToString(" ")}' $label
```

Comment:
I'm not entirely sure what the `\u001b[0m` ANSI code is, but if the `buildozer` command matches `^buildozer '(.*)'\\s+(.*)$`, `ibazel` can automatically run it when it sees it (see https://github.com/bazelbuild/bazel-watcher#output-runner)

Reply:
I copied these codes from Bazel's Java strict deps compiler plugin :-)

Looks like you can set any regex patterns anyway with the output runner, so should be good?